src/cmds/categories/create.ts

```diff
@@ -14,7 +14,7 @@ interface Category {
   type: string;
 }
 
-export interface Options {
```

Comment:
I think the reason I exported these originally was in case we wanted to use them in tests but since we aren't doing that, this is a solid find. No sense in exporting something that isn't used internally, and can't/shouldn't be used externally.

Reply:
yeah it's great. also a great way to identify unused code, it found some good stuff that we can probably clean out of our `api` subpackage in the main codebase